client/src/project/Form.vue

```diff
@@ -317,6 +376,8 @@
         dojos: [],
         numParticipants: 1,
         projectDetails: {},
+        projectQuestions: ['require_power', 'require_wifi', 'is_hazardous', 'other_requirements'],
```

Comment:
If we're hardcoding them here, then there's no point using `hasQuestion` in the template. I guess we either just hardcode them in the template, or we use the questions list from the event instead.

Reply:
In both scenario they are hardcoded, either in the template (hasQuestion({{question}}) or here.
Though, i agree i could have a computed props to get the projectsQuestions.
Is that something that would fit ?

Reply:
Whoops, missed your reply. Sorry! The hasQuestion part though isn't quite the same kind of hard-coding (like you could set up an event that only cares about wifi and the form would only render that).

Yeah, I think a computed property to get it from the event is best.

Reply:
Yeah, no. The reason i've done that was because we know project.questions will contain more questions than this page, and I don't want to default values (from https://github.com/CoderDojo/coolest-platform/pull/42/files/aca0e03694c9a1c86e8cd2c266915c41c8b3ec8c#diff-2bd8453506c3344e974df8bb0b121feaR540 ) to be defaulted to false on save.
This field allow us to "select" the questions which are being used in this page, in comparison to the next one (Extra details)
@DanielBrierton